providers/go-feature-flag/src/main/java/dev/openfeature/contrib/providers/gofeatureflag/bean/BeanUtils.java

```diff
@@ -0,0 +1,20 @@
+package dev.openfeature.contrib.providers.gofeatureflag.bean;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Bean utils.
+ */
+public class BeanUtils {
+
+    private static ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
```suggestion
    private static final ObjectMapper objectMapper = new ObjectMapper();
```

We are not re-affecting the object, so `objectMapper` may be `final`.